### DISCUSSION TITLE
Searchable custom fields

### DISCUSSION THREAD

**Discussion by PieterL75** [netbox-community/netbox#8349] [Ideas]
I would like to be able to search custom fields, using the top-search-bar.
This could be enabled per CustomField, to be searchable, or just for all custom fields in a whole.

Or is there another plan with the CF ?
What in v3.2, when a CF can point to NB model. Will the model containing the CF with the reference also be shown when looking for the referenced object?

ex: Cluster has a CF allowing for IP address objects. IP 10.1.1.1 is assigned to the Cluster.
What happens if I do a search for 10.1.1.1. Will that also give me the Cluster (because of the CF reference) ?

so 2 questions in 1 discussion, but quite related

Pieter

**Comment by Gingy79** [comment]
I would also like search to include custom fields
